src/Parser/RestoDruid/CombatLogParser.js

```diff
@@ -104,8 +107,10 @@ class CombatLogParser extends CoreCombatLogParser {
     darkTitanAdvice: DarkTitanAdvice,
     essenceOfInfusion: EssenceOfInfusion,
     tearstone: Tearstone,
-    t20: T20,
     t19_2set: T19_2Set,
+    t20: T20,
+    t21_2set: T21_2Set,
```

Comment:
I think this should be renamed `t21TwoSet` etc, snake case is creeping in here.

Reply:
All the other set bonus modules are named in this fashion, so I don't think it makes sense to change unless you want a broader refactor. Personally I prefer snake case to '2Two'.

Reply:
Naming that I've used to keep it somewhat consistent and readable was `Tier21_4set`.

Reply:
So change it or keep it as is?